tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
```

Comment:
First, I'm not sure what does that have to do with loading data.
load_anssel doesn't aggregate the second sentences either, does it?

Second, if "hypev" is obsolete, why not name this "hypev"?  I'm not sure
how appropriate "yesno" is e.g. for entrance exam datasets.  It's just
nitpicking, of course, but I don't understand the name change.